task/opm-render-bundles/0.1/opm-render-bundles.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: opm-render-bundles
+spec:
+  description: Create a catalog index and render the provided bundles into it
+  params:
+    - name: binary-image
+      description: Base image in which to use for the catalog image
+      default: registry.redhat.io/openshift4/ose-operator-registry:latest
+    - name: bundle-images
+      description: Comma separated list of bundles to add
+    - name: operator-name
+      description: Name of the Operator
+    - name: operator-version
+      description: Version of the Operator
+    - name: default-channel
+      description: The channel that subscriptions will default to if unspecified
+      default: stable
+  workspaces:
+    - name: source
+      description: Workspace with the source code
+  steps:
+    - name: opm-render-bundles
+      image: "registry.redhat.io/openshift4/ose-operator-registry:latest"
+      workingDir: $(workspaces.source.path)/source
+      securityContext:
+        capabilities:
+          add:
+            - SETFCAP
```

Comment:
Similar to https://github.com/konflux-ci/build-definitions/pull/1210, what we need probably isn't SETFCAP but `runAsUser: 0`

Reply:
updated! Thanks Adam